app/assets/stylesheets/partials/_forms.scss

```diff
@@ -155,3 +155,12 @@ override character count display
         display: block;
     }
 }
+/*
+margin added to item, description and confidential information label on describe your item page
+*/
+.confidential-info-label {
+    margin: 0px 0px 8px 0px;
+}
+.confidential-item-label {
+    margin: 0px;
+}
```

Comment:
What purpose does this new css serve?
Can you use standard library classes instead?